### ISSUE TITLE
Notifications for thanks and usages

### ISSUE THREAD

**Issue by nicolas-raoul** [commons-app/apps-android-commons#2070]
When I get a thank, or when a picture of mine gets used somewhere, a counter is increased in "Achievements". That's hard to see, and most people don't go check more than once a month, I guess.

We should show these two events MUCH more prominently, and I think that a notification is the most natural way to do that.

The notification should not be shown if the image usage results from an edit by me, though (typically from Wikidata). Maybe our server could check that.

As a rule of thumb, as a heavy user, since 2005 I have received 13 thanks and my pictures have been used 229 times, maybe 30% by myself. With https://github.com/commons-app/apps-android-commons/issues/780 the number of thanks could increase a lot, though.

> I would suggest that for WikiMedia [sic]
Commons to be more interesting to a general user
who has access to the internet, he/she would
probably want to contribute more if he or she were
to get something back in return. For example, on
Facebook, people get likes or reactions in the form
of comments which makes them feel good about
contributing, on Quora, you get upvotes and views
which makes you want to write more. If Wikimedia
Commons were to be built on a platform where
people could rate contributions of their friends
and colleagues, it would probably draw more
interest, especially from the millennials. -Man, age
18-21, United Arab Emirates

**Comment by misaochan** [comment]
To be fair, in the new v3.0 UI the counter will likely be shown in Home, loaded whenever the user opens the app. :) However, I still think a notification would be great!

**Comment by chrisdebian** [comment]
Agreed, this would be a great motivator. Has anything been incorporated into the app since 2018?

Chris.

**Comment by nicolas-raoul** [comment]
notification is currently sent for thanks.
notification for usages not implemented yet.
